applications/DEMApplication/python_scripts/materials_assignation_utility.py

```diff
@@ -0,0 +1,112 @@
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.DEMApplication import *
+
+class MaterialsAssignationUtility():
```

Comment:
@maceligueta I strongly suggest to use the `ReadMaterialsUtility` from the Core to have the material files consistent with other apps

Looking at your material files they are almost the same from what is used for e.g. fluid and structure but still slightly different

Since this is a big change anyway I would really look closely if you can make it work also with the Core workflow

Reply:
I agree, most of the apps use that utility like `FluidDynamicsApplication`, an example you can find is in `fluid_solver.py`